x-pack/plugin/fleet/src/internalClusterTest/java/fleet/action/GetGlobalCheckpointsActionIT.java

```diff
@@ -0,0 +1,236 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package fleet.action;
```

Comment:
Can we move this to:

```suggestion
package org.elasticsearch.xpack.fleet.action;
```